lib/logstash/config/mixin.rb

```diff
@@ -113,6 +114,10 @@ def config_init(params)
       instance_variable_set("@#{key}", value)
     end
 
+    @logger.terminal(I18n.t("logstash.plugin.activated_sucessfully", 
```

Comment:
I'm on the fence about this. For folks with long configs, this will generate a lot of information that will likely be deemed "noise".